Authentication Methods Registration Campaign Graph Docs (#12112)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/registrationenforcement.md

---
title: "registrationEnforcement resource type"

> description: "Settings for enforcing registration at sign-in time."

Reviewer: ```suggestion
description: "Enforce registration at sign-in time."
```

author: "mjsantani"
localization_priority: Normal

> ms.prod: "microsoft-identity-platform"

Reviewer: ```suggestion
ms.prod: "identity-and-sign-in"
```

doc_type: resourcePageType
---

# registrationEnforcement resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> The settings available for enforcing registration at sign-in time. This can currently only be used to push users to set up targeted authentication methods (Microsoft Authenticator) using the 'authenticationMethodsRegistrationCampaign`.

Reviewer: ```suggestion
Enforce registration at sign-in time. This can currently only be used to remind users to set up targeted authentication methods (Microsoft Authenticator) using the 'authenticationMethodsRegistrationCampaign`.
```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |authenticationMethodsRegistrationCampaign|[authenticationMethodsRegistrationCampaign](../resources/authenticationmethodsregistrationcampaign.md)|Settings to run campaigns to push users to setup targeted authentication methods.|

Reviewer: ```suggestion
|authenticationMethodsRegistrationCampaign|[authenticationMethodsRegistrationCampaign](../resources/authenticationmethodsregistrationcampaign.md)|Run campaigns to remind users to setup targeted authentication methods.|
```
